hls4ml/templates/quartus/firmware/myproject.cpp

```diff
@@ -42,25 +42,25 @@
                - This is due the HLS Streaming Interfaces; stream cannot be copied (implicitly deleted copy constructor)
 * This distinction is handled in quartus_writer.py
 */
-//hls-fpga-machine-learning instantiate GCC top-level
+// hls-fpga-machine-learning instantiate GCC top-level
 #else
 // Maximum initiation interval, concurrency and frequency for HLS syntheis are defined here
-//hls-fpga-machine-learning insert cpragmas
+// hls-fpga-machine-learning insert cpragmas
 
 /*
-* The top-level function used during HLS Synthesis goes here
-* In a similar manner to GCC, there is a distinction between io_stream & io_parallel
-*/
-//hls-fpga-machine-learning instantiate HLS top-level
-#endif 
-   // If using io_parallel, the output needs to be initialised and returned at the end of this function
-   // If using io_stream, no output is initialised, as it is passed by reference to the top-level function
-   //hls-fpga-machine-learning initialize input/output
+ * The top-level function used during HLS Synthesis goes here
+ * In a similar manner to GCC, there is a distinction between io_stream & io_parallel
+ */
+// hls-fpga-machine-learning instantiate HLS top-level
+#endif
+// If using io_parallel, the output needs to be initialised and returned at the end of this function
+// If using io_stream, no output is initialised, as it is passed by reference to the top-level function
+// hls-fpga-machine-learning initialize input/output
+
+// ****************************************
+// NETWORK INSTANTIATION
+// ****************************************
 
-   // ****************************************
-   // NETWORK INSTANTIATION
-   // ****************************************
+// hls-fpga-machine-learning insert layers
```

Comment:
Hmm, is there a way to exclude templates like this file from pre-commit? These comments are strategically placed to ensure the generated code remains readable. Writers expect this. If the spaces and indentations are removed, it will result in uglier code being emitted.